### PR TITLE
[AUTO-151] Convert values of type IntValue

### PR DESCRIPTION
**Pull request by thomaschaplin**
Fixes https://github.com/trayio/graphql-query-to-json/issues/8

### CODE REVIEW

**Comment by thomaschaplin** [comment]
@rickschubert @BhupenT you may find this PR of interest that fixes https://github.com/trayio/graphql-query-to-json/issues/8

File: src/test/full_functionality.spec.ts [hunk 1]
```diff
@@ -379,6 +415,57 @@ describe("Mutations", () => {
...
+                    },
+                    personal: {
+                        count: true,
+                    },
+                },
+            },
+        })
+    })
+
+    it("Argument is a variable", () => {
```
Comment (thomaschaplin) [inline src/test/full_functionality.spec.ts:442]: This adds code extra code coverage for a line that was missing a test. This is nothing to do with the IntValue change. However, I did choose to use an IntValue in the test.

Adds test coverage for line 86 in index.ts

**Review by rickschubert — APPROVED** [review]
Awesome!

**Comment by rickschubert** [comment]
@thomaschaplin Do you know if we already support float numbers?

**Comment by thomaschaplin** [comment]
> @thomaschaplin Do you know if we already support float numbers?

Good shout!
Not sure, I'll figure it out and raise another PR if not. I'll look at adding some unit tests for it too.

**Review by benifietray — APPROVED** [review]

**Pull request merged**